### PR TITLE
Use long options instead of short ones

### PR DESCRIPTION
**Pull request by clemens-tolboom**
It is a little hard to learn what the options mean.

### CODE REVIEW

File: doc/manual/introduction/quick-start.xml [hunk 1]
```diff
@@ -40,7 +40,8 @@ libxslt-1.1.28
 <step><para>Install some packages from the channel:
 
 <screen>
-$ nix-env -i hello</screen>
+$ nix-env --install hello <replaceable>...</replaceable> </screen>
```
Comment (clemens-tolboom) [inline doc/manual/introduction/quick-start.xml]: Not sure where these replaceable come from :-(

Comment (clemens-tolboom) [inline doc/manual/introduction/quick-start.xml]: https://github.com/NixOS/nix/commit/111d347237d7f313ab9acfafc0de859715c31564

**Comment by clemens-tolboom** [comment]
Hmmm ... thanks @stale to poke me. Will check soon.

**Comment by clemens-tolboom** [comment]
So the quickstart (and more?) are moved from .xml into .md

Commit: e0ea3c82ca9e46359c55c9f716fec016f8d483ea [e0ea3c82c]
Parents: 650ae14ced
Author: Eelco Dolstra <edolstra@gmail.com>
Date: 22 July 2020 at 20:27:23 CEST
Commit Date: 23 July 2020 at 18:26:48 CEST

which got merged in

Commit: b721877b85bbf9f78fd2221d8eb540373ee1e889 [b721877b8]
Parents: ebb8e076eb, 50f46836f3
Author: Eelco Dolstra <edolstra@gmail.com>
Date: 1 September 2020 at 10:31:17 CEST
Committer: GitHub <noreply@github.com>

**Comment by clemens-tolboom** [comment]
Not sure how to trigger tests :-/

**Comment by vcunat** [comment]
I don't think tests are really relevant to such a trivial documentation change.  I'm personally not sure if using long forms is preferable in docs.  Perhaps some of "real" maintainers have a strong opinion on this.

If **I** wanted to learn what the options mean, I'd look into the corresponding man page.  And when I type into console I almost always use the short options.  In scripts I would probably prefer long ones, though...

**Comment by clemens-tolboom** [comment]
> If I wanted to learn what the options mean, I'd look into the corresponding man page. And when I type into console I almost always use the short options. In scripts I would probably prefer long ones, though...

If you don't want to help new users onboarding please do keep the short options in the documentatation.

Making them long would have helped me a lot faster learning new/unfamiliar tools.

**Comment by Ericson2314** [comment]
This seems like a good idea. Start with long, and once you get tired of typing, switch to old.

**Comment by fricklerhandwerk** [comment]
@clemens-tolboom sorry for the long waiting time. Are you still interested in merging this? Please rebase so we can proceed.

**Comment by clemens-tolboom** [comment]
@fricklerhandwerk I pass ... I removed my clone and am not using nix anymore. I rerolled it Feb 13, 2021 hope someone can redo this on

**Comment by Ericson2314** [comment]
Behold, everyone, how we've abandon PRs chasing people off!

May we finally be turning s new leaf.

**Comment by Ericson2314** [comment]
#7864 I have rebased it.

**Pull request closed without merging**